src/components/batch-selection/batch-selection-test.stories.mdx

```diff
@@ -0,0 +1,56 @@
+import { Meta, Story, Preview } from "@storybook/addon-docs";
+import { action } from "@storybook/addon-actions";
+
+import BatchSelection from ".";
+import IconButton from "../icon-button";
+import Icon from "../icon";
+
+<Meta
+  title="Design System/Batch Selection/Test"
+  parameters={{
+    info: { disable: true },
+    chromatic: {
+      disable: true,
+    },
+  }}
+  argTypes={{
+    colorTheme: {
+      options: ["light", "dark", "transparent-base", "transparent-white"],
```

Comment:
These options are different to the prop options specified in the component, they should be the same

Reply:
Good point, thanks!